plutus-core/executables/Common.hs

```diff
@@ -76,7 +76,7 @@ type PlcProg =
 type UplcProg =
   UPLC.Program PLC.Name PLC.DefaultUni PLC.DefaultFun
 
-class Executable p where
+class PlutusProgram p where
```

Comment:
Name bikeshedding: `ProgramLike`? I sometimes use that naming for a typeclass or interface that represents a class of thing that are all kind of similar to X.